Release notes 1.69.x (#8847)
harness/developer-hub: release-notes/continuous-delivery.md

## December

### Version 1.69.x

#### Fixed Issues

> -  Null Pointer Exception was getting thrown while deploying a pipeline with service when no artifact is selected and the primary artifact is provided in the pipeline. This issue is fixed now. (CDS-104756)

Reviewer: ```suggestion
- A null pointer exception was thrown when a pipeline was provided a primary artifact, but no artifact was selected in the service. This issue is fixed now. (CDS-104756)
```

Reviewer: I tried making this note clearer
